dask_cuda/local_cuda_cluster.py

```diff
@@ -90,11 +90,15 @@ class LocalCUDACluster(LocalCluster):
         NOTE: The size is a per worker (i.e., per GPU) configuration, and
         not cluster-wide!
     rmm_managed_memory: bool
-        If True, initialize each worker with RMM and set it to use managed
-        memory. If False, RMM may still be used if `rmm_pool_size` is specified,
+        If ``True``, initialize each worker with RMM and set it to use managed
+        memory. If ``False``, RMM may still be used if `rmm_pool_size` is specified,
         but in that case with default (non-managed) memory type.
         WARNING: managed memory is currently incompatible with NVLink, trying
         to enable both will result in an exception.
+    rmm_log_directory: str
+        Directory to write per-worker RMM log files to. If users want logging
+        from the client or scheduler, that must be set separately. Has no effect
```

Comment:
```suggestion
        Directory to write per-worker RMM log files to, the client and
        scheduler are not logged here. Has no effect
```

Reply:
And again, I don't think the `rmm_pool_size`/`rmm_managed_memory` statement is true here either.